actionview/test/template/form_tag_helper_test.rb

```diff
@@ -88,6 +88,42 @@ def test_check_box_tag_default_checked
     assert_dom_equal expected, actual
   end
 
+  def test_check_box_tag_too_many_arguments
+    assert_raises ArgumentError do
+      check_box_tag "admin", "1", true, false
+    end
+  end
+
+  def test_check_box_tag_checked_kwarg_true
+    actual = check_box_tag "admin", "1", checked: true
+    expected = %(<input id="admin" checked="checked" name="admin" type="checkbox" value="1" />)
```

Comment:
It occurs to me that we should probably specify a non-default value for `value` in these tests, to verify that the `value` positional arg is being properly evaluated.  e.g.:

```suggestion
    actual = check_box_tag "admin", "foo", checked: true
    expected = %(<input id="admin" checked="checked" name="admin" type="checkbox" value="foo" />)
```